subworkflows/detect_rna_swf.nf

```diff
@@ -0,0 +1,27 @@
+/*
+ * Run RNA detection
+*/
+
+include { DETECT_NCRNA } from '../modules/detect_ncrna'
+include { DETECT_TRNA } from '../modules/detect_trna'
+
+workflow DETECT_RNA {
+    take:
+        all_prokka_fnas
```

Comment:
please make 2 changes to this:
- rename all_prokka_fnas -> fnas ( internally this is not linked to prokka )
- do the `all_prokka_fnas.map(it -> [it[1].baseName, it[1]]).join(
                accessions_with_domains
            )` when calling the module

this will simplify publishing this as a generalized / shareble module in our nf-modules repo

Reply:
Tuple renamed in https://github.com/EBI-Metagenomics/genomes-catalogue-pipeline/commit/b86cd9f5a615ba763ba10b9da480b37abd5b8e0f
We decided to leave the second change for later.